mkdocs.yml

```diff
@@ -52,84 +84,87 @@ extra:
-        - Things to Consider in a Multi-Node JanusGraph Cluster: basics/multi-node.md
-        - Transactions: basics/transactions.md
-        - JanusGraph Cache: basics/cache.md
-        - Transaction Log: basics/transaction-log.md
-        - Configuration Reference: basics/configuration-reference.md
-        - Config Example: basics/example-config.md
-        - Common Questions: basics/common-questions.md
-        - Technical Limitations: basics/technical-limitations.md
-    - Index Management:
-        - Indexing for Better Performance: index-management/index-performance.md
-        - Index Lifecycle: index-management/index-lifecycle.md
-        - Reindexing: index-management/index-reindexing.md
-        - Removal: index-management/index-removal.md
-    - Storage Backends:
-        - Introduction: storage-backend/index.md
-        - Apache Cassandra: storage-backend/cassandra.md
-        - Apache HBase: storage-backend/hbase.md
-        - Google Cloud Bigtable: storage-backend/bigtable.md
-        - Oracle Berkeley DB Java Edition: storage-backend/bdb.md
-        - InMemory Storage Backend: storage-backend/inmemorybackend.md
-    - Mixed Index Backends:
-        - Introduction: index-backend/index.md
-        - Search Predicates and Data Types: index-backend/search-predicates.md
-        - Index Parameters and Full-Text Search: index-backend/text-search.md
-        - Field Mapping: index-backend/field-mapping.md
-        - Direct Index Query: index-backend/direct-index-query.md
-        - Elasticsearch: index-backend/elasticsearch.md
-        - Apache Solr: index-backend/solr.md
-        - Apache Lucene: index-backend/lucene.md
-    - Advanced Topics:
-        - Advanced Schema: advanced-topics/advschema.md
-        - Eventually-Consistent Storage Backends: advanced-topics/eventual-consistency.md
-        - Failure & Recovery: advanced-topics/recovery.md
-        - Bulk Loading: advanced-topics/bulk-loading.md
-        - Graph Partitioning: advanced-topics/partitioning.md
-        - Datatype and Attribute Serializer Configuration: advanced-topics/serializer.md
-        - JanusGraph with TinkerPop’s Hadoop-Gremlin: advanced-topics/hadoop.md
-        - Monitoring JanusGraph: advanced-topics/monitoring.md
-        - Migrating from Titan: advanced-topics/migrating-titan.md
-        - Migrating from Thrift: advanced-topics/migrating-thrift.md
-        - JanusGraph Data model: advanced-topics/data-model.md
-        - JanusGraph Bus: advanced-topics/janusgraph-bus.md
+  - Introduction: index.md
+  - Getting Started:
+    - Installation: getting-started/installation.md
+    - Basic Usage: getting-started/basic-usage.md
+    - Gremlin Query Language: getting-started/gremlin.md
+    - Architectural Overview: getting-started/architecture.md
+  - Interactions with JanusGraph:
+    - Transactions: interactions/transactions.md
+    - Search Predicates and Data Types: interactions/search-predicates.md
```

Comment:
This chapter heavily requires mixed index knowledge, which is only covered in the following chapters.

Moreover, some if not all sections in this chapter are related to mixed index only. For example, "Data Type Support" section is written for mixed index only.

Reply:
Yeah they require mixed index but they are part of interacting with JanusGraph. I'm not sure if you 100% need to setup and operate mixed index until you can interact with them. 

I could put a notice in the top of this section. "This section requires mixed index" What do you think?

Reply:
I would like a rewrite or refactor of this chapter, but that can be done later.
"This section requires mixed index" is not accurate because some parts don't really require a mixed index, e.g. Fuzzy search can still be done without having a mixed index.
Probably we can say "Some parts of this section require mixed index".